app/assess/templates/continue_assessment.html

```diff
@@ -13,6 +13,25 @@
     </p>
     {{ logout_partial(sso_logout_url) }}
 </div>
+
+{% if form.errors.get("reason") %}
+<div class="govuk-error-summary" data-module="govuk-error-summary">
+    <div role="alert">
+        <h2 class="govuk-error-summary__title">
+            There is a problem
+        </h2>
+        <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+                <li>
+                    <a href=#{{form.reason.id}}>Reason for continuing assessment</a>
```

Comment:
Can we add the error to the form itself?

I think this would be done here: https://github.com/communitiesuk/funding-service-design-assessment/blob/e4251d9d7376bd95932e374369e94c5c4458efb0/app/assess/forms/continue_application_form.py#L9

See my other comment for an example